package.json

```
@@ -8,6 +8,8 @@
   },
   "scripts": {
     "build": "tsc",
+    "lint": "npx prettier '**/*.{js,jsx,ts,tsx,json}' --check",
+    "lint:fix": "npx prettier '**/*.{js,jsx,ts,tsx,json}' --write",
```

Since `prettier` is already installed as a dev dependency there is no need to use `npx`. I also believe that we can omit the extensions and do the whole directory with all of the Prettier supported extensions instead. I'd also like `--write` to be the default so we'd end up with something like this.

```suggestion
    "lint": "prettier . --write",
```